osu.Game.Rulesets.Mania/Difficulty/Skills/Strain.cs

```diff
@@ -37,31 +37,35 @@ protected override double StrainValueOf(DifficultyHitObject current)
             var maniaCurrent = (ManiaDifficultyHitObject)current;
             double endTime = maniaCurrent.EndTime;
             int column = maniaCurrent.BaseObject.Column;
+            double closestEndTime = 100;
 
             double holdFactor = 1.0; // Factor to all additional strains in case something else is held
             double holdAddition = 0; // Addition to the current note in case it's a hold and has to be released awkwardly
 
             // Fill up the holdEndTimes array
             for (int i = 0; i < holdEndTimes.Length; ++i)
             {
-                // If there is at least one other overlapping end or note, then we get an addition, buuuuuut...
+                // If there is at least one other overlapping end or note, then we get an addition
                 if (Precision.DefinitelyBigger(holdEndTimes[i], maniaCurrent.StartTime, 1) && Precision.DefinitelyBigger(endTime, holdEndTimes[i], 1))
                     holdAddition = 1.0;
```

Comment:
Simplify this since the `1.0` is not important when L66 compares `>0` anyways.
Making this any value `>0` would trigger the same result

`isOverlapping` would be more explicit

```suggestion
                isOverlapping = (Precision.DefinitelyBigger(holdEndTimes[i], maniaCurrent.StartTime, 1) && Precision.DefinitelyBigger(endTime, holdEndTimes[i], 1));
```